index.html

```diff
@@ -0,0 +1,4 @@
+---
+title: Homebrew on Linux
+redirect_to: https://docs.brew.sh/Linuxbrew
```

Comment:
```suggestion
redirect_to: https://docs.brew.sh/Linux
```

Reply:
That page does not exist.

Reply:
See https://github.com/Homebrew/brew/pull/5867#issuecomment-472125846

Reply:
It will be `/Homebrew-on-Linux-and-Windows-10` once https://github.com/Homebrew/brew/pull/5867/ is merged.